frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -0,0 +1,69 @@
+import { browser, by, element, ExpectedConditions as until, $ } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { PVC_STATUS } from '../utils/consts';
+import { PvcType } from '../utils/helpers';
+
+export const selectItemFromDropdown = async (item, dropdownElement) => {
+  await click(dropdownElement);
+  await click($(`#${item}-link`));
+};
+
+// create pvc
+export const namespaceDropdown = $$('[class="pf-c-dropdown__toggle pf-m-plain"]').get(0);
+export const storageclassDropdown = $('#storageclass-dropdown');
+export const inputPVCName = $('#pvc-name');
+export const selectAccessMode = (accessMode) => $(`input[value=${accessMode}]`);
+export const inputPVCSize = $('[name=requestSizeValue]');
+export const sizeUnitsDropdown = $('[data-test-id=dropdown-button]');
+
+// pvc details
+export const pvcName = element('[data-test-id=pvc-name]'));
+export const pvcStatus = element('[data-test-id=pvc-status]'));
+export const pvcSize = element('[data-test-id=pvc-capacity]'));
+export const pvcAccessMode = element('[data-test-id=pvc-access-mode]');
+export const pvcVolumeMode = element('[data-test-id=pvc-volume-mode]');
+export const pvcStorageClass = element('[data-test-id=pvc-storageclass]');
+export const pvcPersistentVolume = element('[data-test-id=persistent-volume]');
+export const actionsButton = $('[data-test-id=actions-menu-button]');
+export const deletePvc = $('[data-test-action="Delete Persistent Volume Claim"]');
+
+// list of PVCs
+export const nameInTable = (name) => $(`a[data-test-id=${name}]`);
+
+export const goToPersistentVolumeClaims = async () => {
+  await sideNavView.clickNavLink(['Storage', 'Persistent Volume Claims']);
+  await crudView.isLoaded();
+};
+
+export const createNewPersistentVolumeClaim = async function(
```

Comment:
I think we should settle on one way of defining functions and stick with it. 
Use either
```
export const fun = async () => {}
```
or
```
export const fun = async function() {};
```

Reply:
For Ceph and Noobaa we follow arrow function styles.

Reply:
I changed everything to arrow function style.